### PR TITLE
11 init solution

### PR DESCRIPTION
**Pull request by nurmukhammadsunatullaev**
Project Created!

### CODE REVIEW

**Comment by nurmukhammadsunatullaev** [comment]
[Init solution](https://trello.com/c/3zVa5Aw9/11-init-solution)

File: SAPex/Controllers/PingController.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace SAPex.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```
Comment (DoniyorJN) [inline SAPex/Controllers/PingController.cs:6]: I prefer the direct name of the controller then abstract name
if you change the name of controller and path also will change

Comment (DoniyorJN) [reply SAPex/Controllers/PingController.cs:6]: [Route("api/[controller]")] => api/ping

**Review by dnovitsky — CHANGES_REQUESTED** [review]
@nurmukhammadsunatullaev 
Please add information into the "readme" file, how to use "swagger" locally.

**Review by DoniyorJN — CHANGES_REQUESTED** [review]

**Review by DoniyorJN — APPROVED** [review]

**Review by dnovitsky — APPROVED** [review]

**Pull request merged**